libs/api/dev/src/lib/api-dev.ts

```diff
@@ -0,0 +1,36 @@
+import { formatApiErrorResponse, validateApiKey } from '@nmit-coursition/api/utils'
+import { type ApiErrorCode, ERROR_LIST } from '@nmit-coursition/api/utils'
+import { Elysia, t } from 'elysia'
+
+const errorResponseType = t.Object({
```

Comment:
**suggestion:** We should move this into `api/utils` as this response type will be shared between many APIs and we want to have it consistent.
For now V1 is not stable yet as we haven't deployed it, so it's safe to alter even `api/v1`.